.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: python
 python:
 - 3.4
-sudo: false
+sudo: true
```

Comment:
I'd prefer not to do this... a) we already use pip for the other dependency and b) this switches the builds from container-based to full VMs which they claim are slower.